tema-css-1/nav.css

```diff
@@ -0,0 +1,12 @@
+.logo {
+    width: 40px;
+    height: 40px;   
+}
+.links {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    background-color: yellow;
+    padding: 10px;
+}
```

Comment:
E foarte buna varianta cu `display: flex`. Good job!
Ideea era sa vezi ca merge alinierea si fara flex cu hint-ul adaugat in descrierea temei (verticle-align: middle) 😄